### ISSUE TITLE
Discontinued as of 1.21

### ISSUE THREAD

**Issue by TheIllusiveC4** [TheIllusiveC4/Curios#427]
This is a (slightly edited) copy of the announcement from the Discord server's `#announcements` channel on 4/10/2024 regarding the state of future Curios development, posted here for archival purposes.

> Hey everyone, it's with a heavy heart that I announce I will not be developing Curios from 1.21 onward. 1.20.6 will be the final port I make.
> 
> I deeply apologize for any inconvenience that this decision may cause. This may also come as a surprise to some of you especially since I was recently making plans for Curios 1.21 and beyond, but I'm sorry to say that these plans are no longer moving forward.
> 
> There wasn't anything specific that spurred this decision, it's just one made after a lot of introspection. I've gradually lost interest in maintaining the project and I've also grown to feel that I'm ill-equipped to handle the challenges of API design in a manner that I find satisfactory. I've only made it this far due to the generous support from the community and a sense of responsibility to continue on but I think it's time for me to stop and refocus on other mods.
> 
> I'm hoping that making this announcement well ahead of 1.21 will give developers time to plan their next steps as I know that many developers rely on Curios and I'm truly sorry that I need to step away from it. I don't plan on officially passing the project to anyone at this time, but Curios is open-source and anyone can continue where I leave off. Or take some of the ideas and build something new.
> 
> I'll continue supporting Curios for 1.20.6 and below for the foreseeable future so anyone on those versions can of course still reach out to me for questions and issues.
> 
> And finally, I just want to thank all those who have used, downloaded, or developed with Curios over these past 5 years since its first release all the way back in 1.13.2. I'm incredibly grateful to everyone and I just hope that Curios has brought something useful to each of you.
> 
> TL;DR I'm no longer developing Curios from 1.21 onward. It's open-source (LGPL 3.0), so anyone can fork or build upon it. Thank you to everyone who used the mod.

For alternatives available on 1.21 and above, please see (list updated as of 7/12/2024):
- [Accessories (alternative API with a Curios compatibility layer)](https://github.com/wisp-forest/accessories)
- [Adorned (Curios fork)](https://github.com/JDKDigital/Adorned)
- [Curios API Continuation (Curios fork)](https://github.com/SSKirillSS/Curios)
